packages/opa/src/opaclient.ts

```diff
@@ -266,6 +338,45 @@ export class OPAClient {
     }
     return Object.fromEntries(items);
   }
+
+  /** `getFilters` is used to translate the policy at the specified path into query filters of
+   * the desired target type, with optional input.
+   * Returns a promise that resolves to an object containing the query filters (key `query`) and optional masks (key `masks`).
+   *
+   * @param path - The path to the policy, without `/v1/compile`: use `filters/include` to translate the policy `data.filters.include`.
+   * @param input - The input to the policy, if needed.
+   * @param opts - Per-request options to control how the policy is translated into query filters, and low-level fetch options.
+   */
+  async getFilters<In extends Input | ToInput>(
+    path: string,
+    input?: In,
+    opts?: FiltersRequestOptions,
+  ): Promise<Filters | MultiFilters> {
+    let inp: Input | undefined = undefined;
+    if (input !== undefined) {
+      if (implementsToInput(input)) {
+        inp = input.toInput();
+      } else {
+        inp = input;
+      }
+    }
+    const target = opts?.target;
```

Comment:
isn't this not true? Don't we assume the target is ucast? I guess that's a business choice, but I'd probably make the assumption to remove some unnecessary complexity.

Reply:
There are two layers here: porcelain, for making it nicer than SE-generated code. The "make magic happen" layer is another one, to follow.

Reply:
(There's no reason to assume that users of `@styra/opa` shouldn't use the TS SDK's porcelain to generate SQL filters. I think we should support making that nice, too.)